docs/deployment/deployment-lite.md

```diff
@@ -19,6 +19,18 @@ only two: a reverse proxy such as Nginx, Traefik or HAProxy and Authelia.
 Documentation for deploying a reverse proxy collaborating with Authelia is available
 [here](./supported-proxies/index.md).
 
+Please note that Authelia only works for websites served over HTTPS because the session cookie
+can only be transmitted over secure connections. Therefore, would you need to generate a
```

Comment:
```suggestion
can only be transmitted over secure connections. Therefore, you need to generate a
```

Reply:
I wanted to let the user know that she "can", not that she "should". What do you think about `Therefore, if you need to generate a...`.

Reply:
Yep just `Therefore, if you need to`.

Reply:
Good.